Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -211,6 +215,9 @@ void PaintableBox::paint_background(PaintContext& context) const
     if (computed_values().border_top().width || computed_values().border_right().width || computed_values().border_bottom().width || computed_values().border_left().width)
         background_rect = absolute_border_box_rect();
 
+    if (computed_values().clip().is_rect() && computed_values().position() == CSS::Position::Absolute)
+        background_rect = computed_values().clip().to_rect().resolved(Paintable::layout_node()).translated(background_rect.location());
+
```

Comment:
Is this correct? This changes the size of the background rect, it does not clip it. This will change the look of gradients and how images are sized/positioned.

Reply:
True, fixed!